### PR TITLE
register as user_not_member (natural person)

### PR DESCRIPTION
**Pull request by r-wasserman**
This is a first PR attempt, covering: 
1. registration as natural person via `user_not_member`, in which only compulsory fields are filled (except checkbox indicated)
2. test data cleanup (see `beforeEach()`) 
3. Missing TODOs are indicated on top of `06_register_member.cy.ts`

### CODE REVIEW

**Review by JoelForamitti — CHANGES_REQUESTED** [review]
The new test didn't pass in the CI:

```  
1) register user_not_member, compulsory fields only (except checkboxes on page 4/5)
       register as natural person:
     CypressError: Timed out retrying after 4000ms: `cy.clear()` failed because this element:

`<input class="p-inputtext p-component" data-v-6c7919b5="">`

is being covered by another element:

`<div class="p-datepicker-header">...</div>`
```

Not sure what the mistake is, but maybe the videos that you can find here under artifact might be of help: https://github.com/MILA-Wien/collectivo-ux/actions/runs/4045446770

File: src/assets/registrationForm.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
                         "extId": "woSqVbVIKwHd",
                         "buttonType": "next"
                     },
-                    "id": "QLeEwzRDCuqz",
+                    "id": "next-page-button",
```
Comment (nanu-c) [inline src/assets/registrationForm.json:51]: Hey r, the id has to be unique. So you can change it to `next-page-button` but the next button has to be `next-page-button-1`

Comment (r-wasserman) [reply src/assets/registrationForm.json]: Hallo @nanu-c, that's a very good point. I took care of it and gave to `<button/>`s a unique id according to the formular page they're on.

Comment (nanu-c) [reply src/assets/registrationForm.json]: perfect!

**Review by JoelForamitti — APPROVED** [review]

**Pull request merged**